fgrpc/dapr.go

```diff
@@ -153,13 +161,84 @@ func (d *DaprGRPCRunnerResults) prepareRequest4PubSub(o *GRPCRunnerOptions) erro
 		} else {
 			d.publishEventRequest.Data = []byte{}
 		}
+	case "bulkpublish":
```

Comment:
@shubham1172 Ideally, bulk publish is not a capability but a method in the pubsub capability/building block. 
Suggest to move bulk publish as method. with capability pubsub. 
For the baseline test, can we make use of the extensions field in the request params to let publish know that it is running a baseline test for bulk publish API?

Reply:
Yup, makes sense! I will do that once I am able to run this locally and test.